### DISCUSSION TITLE
Feature request: Private browsing with Tor

### DISCUSSION THREAD

**Discussion by georg-stone** [zen-browser/desktop#2842] [Ideas]
Tor is an amazing tool for anonymous browsing and I think Zen Browser's private browsing mode could deeply benefit from this. One great example is Brave where you can either open normal private browsing or private browsing with Tor.

**Comment by universish** [comment]
https://github.com/zen-browser/desktop/discussions/2921

**Comment by bahadirdogru** [comment]
This is really good Feature to have.

**Comment by callb4ck** [comment]
It would be nice to have a tor proxy feature similar to either how brave does it or within a workspace or container-like tab.

Of course it shouldn't be intended as a substitute for tor browser with all its privacy preserving features, it would just be a nice quality of life improvement to be able to quickly spin up a tab with a tor proxy.

Perhaps one might even consider blocking or limiting the use of hidden services and just use tor as any normal proxy (even though it isn't)

**Comment by jclaveau** [comment]
Definitely one of the best features of Brave
